cmd/agent/app/flare.go

```diff
@@ -45,6 +52,8 @@ var flareCmd = &cobra.Command{
 			color.NoColor = true
 		}
 
+		cpuProfURL = fmt.Sprintf("http://127.0.0.1:%s/debug/pprof/profile?seconds=%d", config.Datadog.GetString("expvar_port"), profiling)
```

Comment:
The raw URL should be move to a `const` block along with the one for memory.

Reply:
Will that update the URL with the value for the profiling seconds though?

Reply:
If I make them consts, I can't put in the value from the config for the port either with `fmt.Sprintf`

Reply:
Nvm, I moved these to `CreatePerformanceProfile` and moved the method to the `flare` package.